README.md

```diff
@@ -467,7 +468,70 @@ function App() {
 }
 ```
 
-### Theming
+## sx prop
+
+A special `sx` prop lets you apply styles directly to an element. When `sx` prop is specified on a component, Pigment CSS will replace any `sx` prop with `className` and `style` props at build time.
+
+The `sx` prop works on any component, including HTML elements and 3rd-party custom components as long as it is JSX.
+
+### Usage
+
+```js
+<div sx={{ display: 'flex', flexDirection: 'column' }}>
+```
+
+For a React component, it must spread the `className` and `style` to the underlying DOM element, otherwise the styles won't be applied.
```

Comment:
This line doesn't seem applicable. We can do without it because users don't have to care about the spread.

Reply:
```js
// components/Custom.tsx
const AnyComponent = () => {
  return <div>Hello</div>;
};

// App.tsx
<AnyComponent sx={{ fontSize: 12, color: 'red' }} />;
```

How would the above snippet work if the user does not pass props to the element?